src/test/groovy/pl/tomaszdziurko/jvm_bloggers/blogs/BloggersDataFetcherSpec.groovy

```diff
@@ -4,28 +4,28 @@ import spock.lang.Specification
 
 class BloggersDataFetcherSpec extends Specification {
 
-    def "Should store empty optional when url is not valid"() {
+    def "Should not throw exception during fetching when url is not valid"() {
```

Comment:
We've got to test cases here:

* _Should not throw exception during fetching when url is not valid_
* _Should update bloggers data from valid url_

The first one contains _during fetching_ which is nit necessary I think. If you remove it the test cases will better match to each other IMHO:

* _Should not throw exception when url is not valid_
* _Should update bloggers data from valid url_

WDYT?